src/Utils/Finder.php

```diff
@@ -202,6 +202,15 @@ public function getIterator(): \Iterator
 	}
 
 
+	/**
+	 * @return string[]
+	 */
+	public function findPaths(): array
```

Comment:
that's a bad name (there is already findFiles but it does sth completely different), how about `getPaths()`?

Reply:
Renamed.